microsoft-edge/webview2/release-notes/index.md

```diff
@@ -66,9 +67,270 @@ See also:
 ###### SDK-only
 -->
 
+<!-- end of Jul 2024 Release SDK -->
+<!-- end of Jul 2024 Prerelease SDK -->
+<!-- end of template -->
+
+
+<!-- ====================================================================== -->
+## 1.0.2592.51
```

Comment:
in June Release or Prerelease section, add .NET API `CoreWebView2.GetComICoreWebView2`
@victorhuangwq @peiche-jessica